packages/jss-keyframes/src/index.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import defaultJss, {SheetsRegistry, type Jss} from 'jss'
+import warning from 'warning'
+
+type Options = {
+  jss?: Jss,
+  name?: string
+}
+
+let counter = 0
+const registry = new SheetsRegistry()
```

Comment:
in case of react-jss and actually even in case with working with just core, adding another registry complicates things, since one needs to be aware of a separate registry for the keyframes and not forget to ssr it properly?